static/bidder-params/boldwin.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Boldwin Adapter Params",
+    "description": "A schema which validates params accepted by the Boldwin adapter",
+    "type": "object",
+  
+    "properties": {
+      "placementId": {
+        "type": "string",
+        "minLength": 1,
+        "description": "Placement ID"
+      },
+      "endpointId": {
+        "type": "string",
+        "minLength": 1,
+        "description": "Endpoint ID"
+      }
+    },
+    "oneOf": [
+      { "required": ["placementId"] },
+      { "required": ["endpointId"] }
+    ]
+  }
```

Comment:
Yes, this is a copy of Compass.json. Also Boldwin bidder code is identical to Compass adapter code, imp.ext is the same and tests are the same. Like @SyntaxNode mentioned already - should this be an alias?

Reply:
This is not an alias, this is a separate adapter